packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -0,0 +1,199 @@
+Moreover, the d3 libraries versions used are outdated, and support for more data to be stored at die level is needed to implement features offered by the MicroStrategy BI Tool framework.
+
+In addition, the current component contains numerous flaky, hard to track functionalities that is offered by the MicroStrategy plugin wrapper, and until these exists in the core component, it will cause inconsistencies and faulty behavior.
+
+**Visual Design Spec**
+
+Design changes of the existing wafer-map component are out of scope for the current release. We came to this decision, as implementing solutions for all the existing corner cases would be very time-consuming, and we can't afford that. 
+The users are allowed to change the color of the dies within the wafer map, so having a non-default color scheme for the base canvas, rim and background of the wafer map could lead to incorrect behavior.
+A conversation with the design team will be started regarding this topic, but for the current release we will go with the default, existing design.
+[Wafer Map plugin for MicroStrategy Feature](https://dev.azure.com/ni/DevCentral/_workitems/edit/2001490)
+
+### Non-goals
+
+These are the features that would make sense to provide for a visualization widget component such as the wafer map, but will not be part of the current implementation:
+* Color legend
+* Die tooltips
+
+Currently our main goal is to bring in the existing wafer map component into the nimble library, but not to add features.
+
+### Features
+
+We consider now that the most important features of the current component would be:
+* Allowing to set the orientation of the wafer
+* Allowing an option to select specific color schemes for the wafer
+* Allowing to zoom in/out on the wafer-map widget
+* Allowing panning whilst the wafer is zoomed
+* Dynamic size adaptability
+* Scalable for very large number of data points
+
+### Risks and Challenges
+
+The biggest risk and challenge are regarding the rendering of the component when large numbers of data points are present. 
+This widget can and will be used to represent the results of a single wafer and a summary of multiple wafers (data which represents also known as a LOT) in order to recognize patterns in failing sections.
+A LOT contains data about multiple wafers, and one of the functionalities of our main feature is to split up a LOT (and display it) into individual wafers.
+This means that we might be displaying several `nimble-wafer-map` components on a single page each containing several hundred or thousands of data points, each having the individual functionality mentioned in the "Features" section of this spec document.
+We are planning to perform initial benchmarking on this aspect, but not planning to optimize it in the first release.
+### Prior Art/Examples
+
+Please see below a few screenshots regarding the current `wafer-map` component:
+
+![WaferMap Example](file:///Resources/wafer_map_example.png)
```

Comment:
These urls should be relative path urls in order to render on github, ie:

```suggestion
![WaferMap Example](./Resources/wafer_map_example.png)
```

You may have to update other image urls as well